torch_geometric/distributed/local_graph_store.py

```diff
@@ -51,6 +58,48 @@ def _remove_edge_index(self, edge_attr: EdgeAttr) -> bool:
     def get_all_edge_attrs(self) -> List[EdgeAttr]:
         return [self._edge_attr[key] for key in self._edge_index.keys()]
 
+    # starting the partition info related to graph
+
+    def set_num_partitions(self, num_partitions: int) -> bool:
+        self.num_partitions = num_partitions
+        return True
+
+    def set_partition_idx(self, partition_idx: int) -> bool:
+        self.partition_idx = partition_idx
+        return True
+
+    def set_node_pb(
+            self, node_pb: Union[torch.Tensor, Dict[NodeType,
+                                                    torch.Tensor]]) -> bool:
+        self.node_pb = node_pb
+        return True
+
+    def set_edge_pb(
+            self, edge_pb: Union[torch.Tensor, Dict[NodeType,
+                                                    torch.Tensor]]) -> bool:
+        self.edge_pb = edge_pb
+        return True
+
+    def set_partition_meta(self, partition_meta: Dict) -> bool:
+        self.meta = partition_meta
+        return True
+
+    def get_partition_ids_from_nids(self, ids: torch.Tensor,
+                                    ntype: Optional[NodeType] = None):
+        # Get the local partition ids of node ids with a specific node type.
```

Comment:
nit, move this to a function-level doc, with `r"""<doc>"""`

Reply:
done